### PR TITLE
New file

### PR DESCRIPTION
**Pull request by Brijeshthummar02**
issue #3

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -4,31 +4,89 @@
...
+        if command -v docker &>/dev/null; then
+            ENGINE="docker"
+        elif command -v podman &>/dev/null; then
+            ENGINE="podman"
+        else
+            echo "Error: Neither Docker nor Podman is installed."
+            exit 1
+        fi
+    fi
+}
```
Comment (Cereal84) [inline install.sh:32]: The function _check_requirements_ must be moved in _scripts/utils.sh_ file.

**Review by Cereal84 — CHANGES_REQUESTED** [review]

**Comment by Cereal84** [comment]
In anadi.sh are used some commands like $(pwd), those commands are supported in Windows ?
Did you try it on windows machine ?

**Comment by Cereal84** [comment]
Hi @Brijeshthummar02 any news?
Do you need some help?

**Comment by Brijeshthummar02** [comment]
@Cereal84 to be honest i'm not sure if  those commands are supported in Windows ?

**Comment by Cereal84** [comment]
> @Cereal84 to be honest i'm not sure if  those commands are supported in Windows ?

Those things are part of the "Windows Support", otherwise the risk is to have not the support for Windows.

I don't know if those commands are supported (or not) by Windows, unfortunately ilI have no Windows machine.

**Comment by Cereal84** [comment]
@Brijeshthummar02 the support must be added in both install and run script, otherwise even if the installation goes well the user cannot use it anyway.

I haven't Windows installed so I cannot test it. I expect that who is adding the support in some O.S. does the test too in order to see if everything works.

**Comment by Cereal84** [comment]
Close this Pull Request because the user seems not more interested in resolving it.

**Pull request closed without merging**